lib/Service/AccountService.php

```diff
@@ -43,12 +45,17 @@ class AccountService {
 	/** @var IL10N */
 	private $l10n;
 
+	/** @var Manager */
+	private $defaultAccountManager;
+
 	/**
 	 * @param MailAccountMapper $mapper
 	 */
-	public function __construct(MailAccountMapper $mapper, IL10N $l10n) {
+	public function __construct(MailAccountMapper $mapper, IL10N $l10n,
+		Manager $defaultAccountManager) {
```

Comment:
Are the `@param` important?
They were not consistent on that file before the PR anyway, so maybe it is not important... but I'm asking coz I am tempted to change some headers on a file I'm changing on another PR :yum: